lib/onc_certification_g10_test_kit/bulk_export_validation_tester.rb

```diff
@@ -158,7 +163,11 @@ def perform_bulk_export_validation
               'Could not verify this functionality when Bearer Token is required and not provided'
 
       file_list = JSON.parse(status_output).select { |file| file['type'] == resource_type }
-      skip_if file_list.empty?, "No #{resource_type} resource file item returned by server."
+      message = "No #{resource_type} resource file item returned by server."
+      if file_list.empty?
+        omit_if (OMIT_KLASS.include? resource_type), message
+        skip message
+      end
```

Comment:
```suggestion
      if file_list.empty?
        message = "No #{resource_type} resource file item returned by server."
        omit_if (OMIT_KLASS.include? resource_type), message
        skip message
      end
```

Reply:
Is message used outside of this if block? If not, probably should move it inside.